gensidebar.py

```diff
@@ -82,6 +82,7 @@ def write_api(project, desc):
     write_api("ctre", "CTRE Libraries")
     write_api("navx", "NavX Library")
     write_api("rev", "SPARK MAX Library")
+    write_api("revcolor", "REV Color Sensor")
```

Comment:
This is going to create a link that goes nowhere, so let's not do that until we actually have working API documentation.